variant_remapping_tools/tests/test_reads_to_remapped_variants.py

```diff
@@ -59,19 +60,26 @@ def test_process_bam_file(self):
         process_bam_file(bamfile, output_file, out_failed_file, fasta_path, True, 50, summary_file)
 
         expected = [
-            'chr2	48	.	C	A	50	PASS	st=+;rac=.\n',
-            'chr2	48	.	C	T	50	PASS	st=+;rac=.\n',
-            'chr2	98	.	C	CG	50	PASS	st=+;rac=.\n',
+            'chr2	48	.	C	A	50	PASS	st=+\n',
+            'chr2	48	.	C	T	50	PASS	st=+\n',
+            'chr2	98	.	C	CG	50	PASS	st=+\n',
             'chr2	1078	.	A	G	50	PASS	st=+;rac=G-A\n',
-            'chr2	1818	.	AAC	A	50	PASS	st=+;rac=.\n',
-            'chr2	2030	.	A	TCC	50	PASS	st=+;rac=.\n'
+            'chr2	1818	.	AAC	A	50	PASS	st=+\n',
+            'chr2	2030	.	A	TCC	50	PASS	st=+\n'
         ]
         with open(output_file, 'r') as vcf:
             for(i, line) in enumerate(vcf):
                 assert line == expected[i]
             # Expected and Generated VCF have the same number of lines
             assert i+1 == len(expected)
 
+    def test_pass_aligned_filtering(self):
+        left_read = self.mk_read(reference_start=0, reference_end=47, cigartuples=[(0, 47)])
+        right_read = self.mk_read(reference_start=51, reference_end=95,  cigartuples=[(4, 3), (0, 44)])
```

Comment:
Suggest using the `pysam.CSOFT_CLIP`, `pysam.CEQUAL` syntax here also for clarity

Reply:
Add reference to the CONST. Thanks for pointing that out